### DISCUSSION TITLE
No sorting icons appearing on file list

### DISCUSSION THREAD

**Discussion by BAndrews22** [Zettlr/Zettlr#3305] [Q&A] [answered]
I would like to switch my notes over to Zettlr but one essential feature I need is the sorting of my files according to creation date. 
This feature is explained in the Zettlr documentation but when I hover the mouse over the file list or a specific folder, no icons for sorting appear. I suppose that I am overlooking something very simple.
I am using Zettlr 2.2.3 on Windows 10.

Thanks!
Bob

**Comment by nathanlesage** [comment, marked as answer]
That is likely an outdated information. You can sort your folders by right-clicking them and choosing properties. Additionally, there are a few settings in the preferences that control, e.g., whether the time-sorter sorts by creation time or modification time.

**Comment by BAndrews22** [comment]
Thanks a mil for this nathanlesage! That does indeed work. I was too stubbornly looking for these symbols...
